hawk/src/main/java/com/orhanobut/hawk/HawkBuilder.java

```diff
@@ -36,6 +37,7 @@
   private Parser parser;
   private Encryption encryption;
   private Callback callback;
+  private Status status;
```

Comment:
can be final and we can move it to the top of the class, I usually separate modifier and put them together for the sake of consistency